src/bundle/Resources/public/js/alloyeditor/src/buttons/ez-btn-tablecolumn.js

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import AlloyEditor from 'alloyeditor';
+
+export default class EzBtnTableColumn extends AlloyEditor.ButtonTableColumn {
+    static get key() {
+        return 'eztablecolumn';
+    }
+
+    /**
+     * Lifecycle. Renders the UI of the button.
+     *
+     * @method render
+     * @return {Object} The content which should be rendered.
+     */
+    render() {
+        let buttonCommandsList;
+        let buttonCommandsListId;
+
+        if (this.props.expanded) {
```

Comment:
Same as above. The similar suggestion applies to all the situations similar to this one.